packages/redis/test/Adaptor.test.js

```diff
@@ -621,3 +622,82 @@ describe('jSet', () => {
     await jSet('animals:1', { name: 'mammoth' })(state);
   });
 });
+
+describe('mSet', () => {
+  const mSetClient = {
+    json: {
+      mSet: async (...args) => {
+        expect(args).to.deep.equal([
+          'animals:1', '$', JSON.stringify({ name: 'mammoth' }),
```

Comment:
Ooh i see, so if i understand correctly `mSet()` can take an array of objects or array of strings. Eg `['key', 'path', 'value', 'key', 'path', 'value']`